clusterNode.py

```diff
@@ -130,19 +130,19 @@ def setup_vm(self, iso_or_image_path: str) -> host.Result:
         else:
             network = "bridge=virbr0"
         cmd = f"""
-        virt-install
-            --connect qemu:///system
-            -n {self.config.name}
-            -r {self.config.ram}
-            --cpu host
-            --vcpus {self.config.cpu}
-            --os-variant={self.config.os_variant}
-            --import
-            --network {network},mac={self.config.mac}
-            --events on_reboot=restart
-            {cdrom_line}
-            --disk path={self.config.image_path}
-            {append}
+        virt-install                                    \\
+            --connect qemu:///system                    \\
+            -n {self.config.name}                       \\
+            -r {self.config.ram}                        \\
+            --cpu host                                  \\
+            --vcpus {self.config.cpu}                   \\
+            --os-variant={self.config.os_variant}       \\
+            --import                                    \\
+            --network {network},mac={self.config.mac}   \\
+            --events on_reboot=restart                  \\
+            {cdrom_line}                                \\
+            --disk path={self.config.image_path}        \\
+            {append}                                    \\
         """
```

Comment:
add `cmd = cmd.replace("\n", "")` instead of hardcoding `\\` with a bunch of spaces.

Reply:
then it should be `shlex.join(shlex.split(cmd))`.

Reply:
Why not cmd.replace("\n", "")?

Reply:
because the command line will be logged, then it has a bunch of whitespaces that result is an overly long line and unusual spacing.

Reply:
We log this for debugging purposes. We want to be able to copy-paste from the logs and debug a failure outside of cda. Will this cause the logs to print something else?

Reply:
no.